core/domain/auth_jobs_one_off.py

```diff
@@ -110,10 +111,19 @@ def map(item):
 
         # The item must be an instance of auth_models.FirebaseSeedModel.
         elif item.id == auth_models.ONLY_FIREBASE_SEED_MODEL_ID:
-            for user in firebase_auth.list_users().iterate_all():
-                auth_id, assoc_info = (
-                    user.uid, (cls.FIREBASE_ACCOUNT_KEY, (None, user.disabled)))
-                yield (auth_id, assoc_info)
+            page = firebase_auth.list_users(max_results=1000)
+            user_batch = []
+            while page is not None:
+                user_batch[:] = page.users # NOTE: Avoids allocating a new list.
+                if not user_batch:
+                    break
+                for user in user_batch:
+                    auth_id, assoc_info = (
+                        user.uid,
+                        (cls.FIREBASE_ACCOUNT_KEY, (None, user.disabled)))
+                    yield (auth_id, assoc_info)
+                logging.info('Page processed: %d Firebase accounts')
```

Comment:
Just noting that this wasn't useful because we get a lot of info logs, so they get buried amongst the mess ("warning" would have been better). 

I did find them eventually. Got a lot of "Page processed: %d Firebase accounts" :P

Might want to fix these issues in a separate PR, but I'm approving since it seems to work and we kind of want it in this release.

Reply:
Hmm I'm inclined to delete it entirely, then. WDYT?

Reply:
Sure, sg!

Reply:
Done by https://github.com/oppia/oppia/pull/12441